config/sync/views.settings.yml

```diff
@@ -10,7 +10,7 @@ ui:
     performance_statistics: false
     preview_information: true
     sql_query:
-      enabled: false
+      enabled: true
```

Comment:
Only affect the views admin UI. Just a dev QOL thing.

Reply:
Should be default, IMO.